build/source/netcdf/def_output.f90

```diff
@@ -317,18 +319,40 @@ subroutine def_variab(ncid,iFreq,hruDesire,timeDesire,metaData,ivtype,err,messag
    call netcdf_err(err,message); if (err/=0) return
 
    ! add parameter description
-   err = nf90_put_att(ncid,iVarId,'long_name',trim(metaData(iVar)%vardesc))
+   catName = trim(metaData(iVar)%vardesc)//' ('//trim(get_statName(iStat))
+   catName = trim(catName)//')' 
+   err = nf90_put_att(ncid,iVarId,'long_name',trim(catName))
    call netcdf_err(err,message); if (err/=0) return
 
    ! add parameter units
-   err = nf90_put_att(ncid,iVarId,'units',trim(metaData(iVar)%varunit))
+   catName = trim(metaData(iVar)%varunit) 
+   if (iStat==iLookStat%totl) then
+    ! make sure that the units of this varaible allow for integration
+    if ((index(catName,'s-1')<=0).and.(index(catName,'s-2')<=0).and.(index(catName,'W m-2')<=0)) then 
+     err=20
+     message=trim(message)//'trying to integrate a non-time variable: '//trim(metaData(iVar)%varName)//' - units: '//trim(catName)
+     return
+    endif
```

Comment:
Since it is not affect the model run, probably just throw a warning message instead of raising an error and stopping the run.